PKGBUILD

```
@@ -46,7 +46,7 @@ build() {
   cmake ${srcdir}/${_dir} \
         -DCATKIN_BUILD_BINARY_PACKAGE=ON \
         -DCMAKE_INSTALL_PREFIX=/opt/ros/noetic \
-        -DPYTHON_EXECUTABLE=/usr/bin/python2 \
+        -DPYTHON_EXECUTABLE=/usr/bin/python3 \
```

This should be:

```suggestion
        -DPYTHON_EXECUTABLE=/usr/bin/python \
```

so as to keep it as generic as possible